code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -337,7 +337,8 @@ var/list/datum/stack_recipe/silver_recipes = list ( \
 
 var/list/datum/stack_recipe/clown_recipes = list ( \
 	new/datum/stack_recipe("bananium floor tile", /obj/item/stack/tile/mineral/clown, 1, 40, 20), \
-	new/datum/stack_recipe/dorf("dorf chair",/obj/structure/bed/chair, 20, one_per_turf = 1, on_floor = 1, inherit_material = TRUE, gen_quality = TRUE),\
+	new/datum/stack_recipe("Clownnonball", /obj/item/cannonball/bananium, 20, time = 3 SECONDS, on_floor = 1), \
```

Comment:
this isn't a proper noun

also not sure about double n clownnon vs clownon but that's just taste